web/src/dropdown_widget.js

```diff
@@ -341,6 +342,12 @@ export class DropdownWidget {
                     show_colored_icon: true,
                 }),
             );
+        } else if (option.user) {
+            $(this.widget_value_selector).html(
+                `<div class='pill-container pill-container-btn'>${render_input_pill(
```

Comment:
We do not have existing template that includes `pill-container`, so have done this. Might be better if we use a handlebar template directly in `.html()`.